src/utils/use-news-handlers.js

```diff
@@ -13,7 +22,83 @@ const useNewsHandlers = () => {
   const [enText, enSetText] = useState('');
   const [enTitle, enSetTitle] = useState('');
 
+  const [tabsValue, setTabsValue] = useState(0);
+  const [checkboxes, setCheckboxes] = useState(checkboxStates);
+  const [preferredLanguages, setPreferredLanguages] = useState([]);
+
+  const handleTabsChange = (event, newValue) => {
+    setTabsValue(newValue);
+  };
+
+  const handleCheckboxChange = (event) => {
+    setCheckboxes({ ...checkboxes, [event.target.name]: event.target.checked });
+  };
+
+  const languageCheckboxes = languages.map((lang, index) => (
+    <FormControlLabel
+      key={index}
+      control={
+        <Checkbox
+          checked={checkboxes[`${lang}`]}
```

Comment:
Simplify to `checked={checkboxes[lang]}`

Reply:
Fixed!